Makefile

```
@@ -1,16 +1,21 @@
-CFLAGS:= -Wall -Wextra -Wfloat-conversion -Wno-unused-parameter -Wstrict-aliasing=1 -Wsign-conversion -I. -Isw_i2c
+CFLAGS:= -Wall -Wextra -Wfloat-conversion -Wno-unused-parameter -Wstrict-aliasing=1 \
+	-Wsign-conversion -Icommon -Ii2c -Ii2c/hw -Ii2c/sw -Ishdlc
```

```suggestion
	-Wsign-conversion -Icommon -Ii2c -Ii2c/sw -Ishdlc
```

I think HW is not needed since there's no header there (SW has the SWi2c HAL as a header) - it wasn't there before either

---

Yeah, I know but i just added it for consistency, then it's symmetrical. Should we remove it?

---

Yes remove